### ISSUE TITLE
The GIF attached to an issue is broken after the issue has been reported 

### ISSUE THREAD

**Issue by cipriansv** [webcompat/webcompat.com#2719]
**Steps to reproduce:**
1. Navigate to: https://webcompat.com/issues/new
2. Report an issue with an attached GIF.
3. Observe the GIF rendering.

**Expected Behavior:**
The GIF must be displayed with no rendering issues.

**Actual Behavior:**
The GIF is broken. See issue #21981.

**Notes:**
1. The issue is reproducible on Chrome 70.0.3538.110

**Watchers:**
@softvision-oana-arbuzov
@softvision-sergiulogigan
@cipriansv

sv;

**Comment by magsout** [comment]
@softvision-sergiulogigan oh got it. 

Same gif, but the first was filled directly withing the webcompat form and the second within a comment on Github, right?

**Comment by softvision-sergiulogigan** [comment]
@magsout yup

**Comment by karlcow** [comment]
Probably related to #1024

I need to check if pillow has improved on the animated GIF side of things.

**Comment by karlcow** [comment]
We are currently using `Pillow==5.2.0`

The current version is [5.4.1](https://pillow.readthedocs.io/en/stable/releasenotes/5.4.1.html)

Ah I see that in [4.1.0](https://pillow.readthedocs.io/en/stable/releasenotes/4.1.0.html?highlight=gif#changes-to-gif-handling-when-saving) something was changed with regards to GIF and Palettes.

The code is here.
https://github.com/webcompat/webcompat.com/blob/2e90cb23706d4136e4d341622810c179b1b423ba/webcompat/api/uploads.py#L96-L124

**Comment by karlcow** [comment]
```html
<a
  href="https://webcompat.com/uploads/2018/11/93e4a08e-aa43-4bd5-85b9-f50e3a73cb7b.gif"
  rel="nofollow"
  ><img
    src="https://camo.githubusercontent.com/e14dfdd6cf740863938dd16731ed7df0882ee4ff/68747470733a2f2f776562636f6d7061742e636f6d2f75706c6f6164732f323031382f31312f39336534613038652d616134332d346264352d383562392d6635306533613733636237622d7468756d622e676966"
    alt="Screenshot Description"
    data-canonical-src="https://webcompat.com/uploads/2018/11/93e4a08e-aa43-4bd5-85b9-f50e3a73cb7b-thumb.gif"
    style="max-width:100%;"
/></a>

```

how come the image is on GitHub… It is supposed to be on webcompat.com.
* https://webcompat.com/uploads/2018/11/93e4a08e-aa43-4bd5-85b9-f50e3a73cb7b-thumb.gif
* https://webcompat.com/uploads/2018/11/93e4a08e-aa43-4bd5-85b9-f50e3a73cb7b.gif

I need to test with some gifs.

**Comment by karlcow** [comment]
ok I checked most of the animated gif uploaded in 2018. All of them have a broken palette of colors.
There is something to fix there (if we can).

**Comment by karlcow** [comment]
Yeah still not solved on Pillow side. https://github.com/python-pillow/Pillow/issues/2893

**Comment by karlcow** [comment]
In fact a lot of issues :)
https://github.com/python-pillow/Pillow/issues?q=is%3Aissue+is%3Aopen+gif+label%3A%22Palette+Issue%22
